addons/mrp/report/mrp_report_bom_structure.py

```diff
@@ -258,7 +266,7 @@ def _get_component_data(self, parent_bom, parent_product, warehouse, bom_line, l
         if not ignore_stock:
             # Useless to compute quantities_info if it's not going to be used later on
             quantities_info = self._get_quantities_info(bom_line.product_id, bom_line.product_uom_id, product_info, parent_bom, parent_product)
-        availabilities = self._get_availabilities(bom_line.product_id, line_quantity, product_info, bom_key, quantities_info, level, ignore_stock)
+        availabilities = self._get_availabilities(bom_line.product_id, line_quantity, product_info, parent_bom, quantities_info, level, ignore_stock)
```

Comment:
Given that in the end, it's still the bom/parent_bom's `id` that is used as key, is there a reason for this change?